ui/docs/manual/access-control/third-party.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Third-Party Sign-In
+order: 30
+---
+
+When Taskcluster is integrated with other web applications, it is useful for those web applications to be able to make REST API calls on behalf of a user.
+The third-party sign-in process is being [rewritten to use an OAuth2 flow](https://github.com/taskcluster/taskcluster-rfcs/blob/master/rfcs/0147-third-party-login.md) and will be documented here.
+
+The current implementation is described [here](/docs/manual/using/integration).
```

Comment:
From [`/docs/manual/using/integration/README.md`](https://github.com/taskcluster/taskcluster/blob/6704c2694a011b35de304abab1f1161e3ba64d2c/ui/docs/manual/using/integration/README.md):

> For example, the Taskcluster tools site is built to be generally useful and to provide a clear view of the Taskcluster system, but may not be suited to providing the information your developers need.

Maybe we should make mention of the new UI instead of Taskcluster tools.